src/screens/FeedScreen/index.js

```diff
@@ -266,7 +265,7 @@ const FeedScreen = (props) => {
         refreshAnonymous={onDeleteBlockedPostCompletedHandle}
         screen="screen_feed"
       />
-    </SafeAreaProvider>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the closing tag should be `</SafeAreaProvider>` instead of `</View>`.
```diff
- </View>
+ </SafeAreaProvider>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->